### DISCUSSION TITLE
Tests passes locally but when I push "Run tests without waiting on API" fails.

### DISCUSSION THREAD

**Discussion by AshwiniJatti** [DevSkillsHQ/discusssions#55] [Help] [answered]
When I run tests locally it shows 
![image](https://github.com/DevSkillsHQ/discusssions/assets/95354234/422e439a-b91a-4033-a23a-b3a7ba8099d4)
but it is showing  "AssertionError: Timed out retrying after 4000ms: Expected to find element: `[data-type=title]`, but never found it."
![image](https://github.com/DevSkillsHQ/discusssions/assets/95354234/83844331-9a33-4e61-9d49-ba5b558b7060)

**Comment by ludvig** [comment, marked as answer]
Hello @AshwiniJatti, looking at your pull request it seems there are some missing files to complete the assessment, please make sure you commit and push all your local changes (specifically files such as `*.tsx / *.ts` etc). Currently only test files/config files has been pushed.

Thank you!

**Reply by AshwiniJatti** [reply]
Thank you @ludvig. It's working fine now. All checks Passed.
